src/components/ux-alert/ux-alert.hbs

```diff
@@ -0,0 +1,8 @@
+<div data-alert class="alert-box {{class}}">
+	{{#if isDataModel}}
+		{{{text}}}
+	{{else}}
+		{{yield}}
+	{{/if}}
+	<a href="#" class="close">&times;</a>
```

Comment:
Maybe you could leverage .teardown()

http://docs.ractivejs.org/latest/ractive-teardown